lib/player.js

```diff
@@ -7026,7 +7026,7 @@ shaka.Player.TYPICAL_BUFFERING_THRESHOLD_ = 0.5;
  * @define {string} A version number taken from git at compile time.
  * @export
  */
-shaka.Player.version = 'v4.3.0-uncompiled';
+shaka.Player.version = 'v4.4.0-uncompiled';
```

Comment:
Why?

Reply:
I don't know how this happened. I'll revert.